packages/front-end/pages/admin/OrgsTable.tsx

```diff
@@ -0,0 +1,156 @@
+      </button>
+      <p>Click an organization name below to switch to it.</p>
+      <div className="mb-2 row align-items-center">
+        <div className="col-auto">
+          <form
+            className="d-flex form form-inline"
+            onSubmit={(e) => {
+              e.preventDefault();
+              setPage(1);
+              loadOrgs(1, query);
+            }}
+          >
+            <Field
+              label="Search:"
+              labelClassName="mr-2"
+              value={query}
+              onChange={(e) => setQuery(e.target.value)}
+              type="search"
+            />
+            <div>
+              <button type="submit" className="btn btn-primary ml-2">
+                <FaSearch />
+              </button>
+            </div>
+          </form>
+        </div>
+        <div className="col-auto">
+          <span className="text-muted">
+            {numberFormatter.format(total)} matching organization
+            {total === 1 ? "" : "s"}
+          </span>
+        </div>
+      </div>
+      {error && <div className="alert alert-danger">{error}</div>}
+      <div className="position-relative">
+        {loading && <LoadingOverlay />}
+        <table className="table appbox">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Owner</th>
+              <th>Created</th>
+              <th>Id</th>
+              {!isCloud() && <th>External Id</th>}
+              <th></th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {orgs.map((o) => (
+              <OrganizationRow
+                organization={o}
+                showExternalId={!isCloud()}
+                key={o.id}
+                current={o.id === orgId}
+                onUpdate={() => {
+                  loadOrgs(page, query);
+                }}
+                switchTo={(org) => {
+                  if (setOrgId) {
+                    setOrgId(org.id);
+                  }
+                  if (setSpecialOrg) {
+                    setSpecialOrg(org);
+                  }
+                }}
+              />
+            ))}
+          </tbody>
+        </table>
+        <Pagination
+          currentPage={page}
+          numItemsTotal={total}
+          perPage={50}
+          onPageChange={(page) => {
+            setPage(page);
+            loadOrgs(page, query);
+          }}
+        />
+      </div>
+      {orgModalOpen && (
+        <CreateOrganization
+          showExternalId={!isCloud()}
+          onCreate={() => {
+            loadOrgs(page, query);
+          }}
+          close={() => setOrgModalOpen(false)}
+        />
+      )}
+      {!isCloud() && isMultiOrg() && (
+        <div className="divider border-top mt-3">
+          <OrphanedUsersList
+            mutateUsers={() => {
+              loadOrgs(page, query);
+            }}
+            numUsersInAccount={0}
+            enableAdd={false}
+          />
+        </div>
+      )}
```

Comment:
I think index.tsx is maybe a better place for this logic.

Reply:
The counter is that it's better placed here since it's tied to the page and query state vars of the orgs table. Let me know what you think

Reply:
From what I can tell that is a mistake.  Since `enableAdd` is `false`, `mutateUsers` which is called on add should never get called.  There is no need to update the orgs members if an orphaned user just gets deleted.  I'd still move it and get rid of mutateUsers doing anything.

Reply:
Good points; updated and moved as requested